### DISCUSSION TITLE
How to configure WebService

### DISCUSSION THREAD

**Discussion by groiso** [YetiForceCompany/YetiForceCRM#15843] [How to]
Hi,

Standard webservice API need to use this doc https://doc.yetiforce.com/api/ changes anything?

If you show me exactly where the error is, I can try with Hostagtor support to make the change or look for another Server, already questioning whether or not this YetiForce requirement exists.

Thanks

**Comment by groiso** [comment]
Hi, @mariuszkrzaczkowski 

It seems that the problem is not the server, I believe it must be a bug in YetiForce or a configuration that is not easily detected and resolved.

I did an installation on localhost, on a PC. All requirements were met and it still didn't install.

I didn't understand how to configure the web service. Just fill in the name and password fields and choose "Standard webservice" and copy the default key and URL, Login and password. I don't see anything beyond this to configure the Webservice.

Thanks

**Reply by mariuszkrzaczkowski** [reply]
> Hi, @mariuszkrzaczkowski
> It seems that the problem is not the server, I believe it must be a bug in YetiForce or a configuration that is not easily detected and resolved.

We have launched the portal many times, everything works properly, you only need to have the correct server configuration and follow the documentation.
If you can't handle it, go to someone who will do it for you

> I did an installation on localhost, on a PC. All requirements were met and it still didn't install.
> I didn't understand how to configure the web service. Just fill in the name and password fields and choose "Standard webservice" and copy the default key and URL, Login and password. I don't see anything beyond this to configure the Webservice.
> Thanks

If you included all the information I wrote, we could help you and write where you have problems but you did not do it, e.g. writing with the logs empty is something impossible

**Reply by groiso** [reply]
Hi, @mariuszkrzaczkowski 

Follow the logs in compressed file. It includes everything that came within the cache and root directory of the system.
[Logs.zip](https://github.com/YetiForceCompany/YetiForceCRM/files/8403445/Logs.zip)
These are the logs I have, if you need others please let me know, thanks.

**Comment by mariuszkrzaczkowski** [comment]
> Hi,
> Standard webservice API need to use this doc https://doc.yetiforce.com/api/ changes anything?

No, because your server needs configuration repair

Do not create duplicates #15820, nobody wants to answer your questions because you need an administrator and a pro-grammer who will help you in starting and configuring CRM and the Portal.
You have basic problems, configuration + lack of proper knowledge 

> If you show me exactly where the error is, I can try with Hostagtor support to make the change or look for another Server, already questioning whether or not this YetiForce requirement exists.

I have already written many times where you have a problem

**Reply by groiso** [reply]
Hi, @mariuszkrzaczkowski 

Above has attached a system logs file, I thought it best to zip. Sorry for anything.
Follow the logs in compressed file. It includes everything that came within the cache and root directory of the system.
[Logs.zip](https://github.com/YetiForceCompany/YetiForceCRM/files/8403445/Logs.zip)
